typescript/sdk/src/metadata/chainMetadataTypes.ts

```diff
@@ -194,6 +194,15 @@ export const ChainMetadataSchemaObject = z.object({
     .boolean()
     .optional()
     .describe('Whether the chain is considered a testnet or a mainnet.'),
+  index: z
+    .object({
+      from: z
+        .number()
+        .optional()
+        .describe('The block to start any indexing from.'),
```

Comment:
Expecting more fields here eventually?

Reply:
I think we will actually - at least on the agents side we plan to add some alternate indexing modalities and this would be the place it'd be configured